lib/abide_dev_utils/cli/xccdf.rb

```diff
@@ -16,6 +16,38 @@ def initialize
         add_command(CmdParse::HelpCommand.new, default: true)
         add_command(XccdfToHieraCommand.new)
         add_command(XccdfDiffCommand.new)
+        add_command(XccdfGenMapCommand.new)
+      end
+    end
+
+    class XccdfGenMapCommand < CmdParse::Command
+      CMD_NAME = 'gen-map'
+      CMD_SHORT = 'Generates mappings from XCCDF files'
+      CMD_LONG = 'Generates mappings for CEM modules from 1 or more XCCDF files as YAML'
+      def initialize
+        super(CMD_NAME, takes_commands: false)
+        short_desc(CMD_SHORT)
+        long_desc(CMD_LONG)
+        options.on('-b [TYPE]', '--benchmark-type [TYPE]', 'XCCDF Benchmark type CIS by default') { |b| @data[:type] = b }
+        options.on('-o [PATH]', '--out-file-path [PATH]', 'Path to save files data/mappings by default') { |d| @data[:path] = d }
```

Comment:
This may not be the best option name to use. `-o` / `--out-file` are used throughout the CLI specifically for configuring the path / name of a single file. This should probably be changed to a more descriptive option name.